{"level":60,"time":1641500000492,"pid":45,"hostname":"b47cfd132c8b","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641506276403,"pid":45,"hostname":"60d9fd55907f","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641506836593,"pid":30,"hostname":"42c98d15f3df","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641506896732,"pid":32,"hostname":"d0da96fb6353","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641507590621,"pid":31,"hostname":"882f78e702eb","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641509175798,"pid":31,"hostname":"3ee635aaadaf","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641512529708,"pid":31,"hostname":"f240ee77ce19","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641512822908,"pid":20,"hostname":"bfc249a6d174","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641512848483,"pid":20,"hostname":"13de461afa40","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641512876440,"pid":30,"hostname":"3b70c604dab7","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641513014552,"pid":30,"hostname":"0d5721163bc9","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641514178192,"pid":50,"hostname":"0d5721163bc9","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641514860106,"pid":62,"hostname":"0d5721163bc9","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641514943582,"pid":74,"hostname":"0d5721163bc9","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641515036274,"pid":86,"hostname":"0d5721163bc9","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641515267351,"pid":98,"hostname":"0d5721163bc9","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641515281667,"pid":110,"hostname":"0d5721163bc9","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641515665626,"pid":44,"hostname":"afa81588873f","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:27:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641516161382,"pid":19,"hostname":"7b168350aafe","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:28:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641516170928,"pid":19,"hostname":"7b168350aafe","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641516180023,"pid":17,"hostname":"7b168350aafe","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641557859947,"pid":19,"hostname":"0d9f44b06f3e","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641557865293,"pid":20,"hostname":"0d9f44b06f3e","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641557871489,"pid":18,"hostname":"0d9f44b06f3e","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641559077437,"pid":20,"hostname":"0d9f44b06f3e","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641559082796,"pid":18,"hostname":"0d9f44b06f3e","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641559088368,"pid":19,"hostname":"0d9f44b06f3e","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641559094333,"pid":18,"hostname":"0d9f44b06f3e","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641568077312,"pid":102,"hostname":"693034cdd6ca","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:27:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641568202763,"pid":37,"hostname":"867d9e6f3451","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:27:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641568329170,"pid":60,"hostname":"86b82d56adae","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641568390159,"pid":48,"hostname":"b4378b1e4110","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:27:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641568589466,"pid":49,"hostname":"0802e6bddf11","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:27:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641568653018,"pid":49,"hostname":"7429b038e808","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:27:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641568936015,"pid":20,"hostname":"f9dda87c23eb","msg":"uncaughtException event","err":{"type":"TypeError","message":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js","stack":"TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js\n    at new NodeError (node:internal/errors:371:5)\n    at watch (node:fs:2245:11)\n    at Object.<anonymous> (/home/app/dist/server.js:29:16)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47","code":"ERR_FEATURE_UNAVAILABLE_ON_PLATFORM"},"msg":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js"}
{"level":60,"time":1641568941680,"pid":19,"hostname":"f9dda87c23eb","msg":"uncaughtException event","err":{"type":"TypeError","message":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js","stack":"TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js\n    at new NodeError (node:internal/errors:371:5)\n    at watch (node:fs:2245:11)\n    at Object.<anonymous> (/home/app/dist/server.js:29:16)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47","code":"ERR_FEATURE_UNAVAILABLE_ON_PLATFORM"},"msg":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js"}
{"level":60,"time":1641568948101,"pid":19,"hostname":"f9dda87c23eb","msg":"uncaughtException event","err":{"type":"TypeError","message":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js","stack":"TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js\n    at new NodeError (node:internal/errors:371:5)\n    at watch (node:fs:2245:11)\n    at Object.<anonymous> (/home/app/dist/server.js:29:16)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47","code":"ERR_FEATURE_UNAVAILABLE_ON_PLATFORM"},"msg":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js"}
{"level":60,"time":1641568954338,"pid":19,"hostname":"f9dda87c23eb","msg":"uncaughtException event","err":{"type":"TypeError","message":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js","stack":"TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js\n    at new NodeError (node:internal/errors:371:5)\n    at watch (node:fs:2245:11)\n    at Object.<anonymous> (/home/app/dist/server.js:29:16)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47","code":"ERR_FEATURE_UNAVAILABLE_ON_PLATFORM"},"msg":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js"}
{"level":60,"time":1641568960977,"pid":17,"hostname":"f9dda87c23eb","msg":"uncaughtException event","err":{"type":"TypeError","message":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js","stack":"TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js\n    at new NodeError (node:internal/errors:371:5)\n    at watch (node:fs:2245:11)\n    at Object.<anonymous> (/home/app/dist/server.js:29:16)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47","code":"ERR_FEATURE_UNAVAILABLE_ON_PLATFORM"},"msg":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js"}
{"level":60,"time":1641568968677,"pid":18,"hostname":"f9dda87c23eb","msg":"uncaughtException event","err":{"type":"TypeError","message":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js","stack":"TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js\n    at new NodeError (node:internal/errors:371:5)\n    at watch (node:fs:2245:11)\n    at Object.<anonymous> (/home/app/dist/server.js:29:16)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47","code":"ERR_FEATURE_UNAVAILABLE_ON_PLATFORM"},"msg":"The feature watch recursively is unavailable on the current platform, which is being used to run Node.js"}
{"level":60,"time":1641569238708,"pid":17,"hostname":"4337c21e542d","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:32:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641569244985,"pid":19,"hostname":"4337c21e542d","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/dist/server.js:32:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"},"msg":"Oops!"}
{"level":60,"time":1641570111052,"pid":55,"hostname":"7677ae9f3529","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module._compile (/home/app/node_modules/source-map-support/source-map-support.js:568:25)\n    at Module.m._compile (/tmp/ts-node-dev-hook-19040724413613908.js:69:33)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at require.extensions.<computed> (/tmp/ts-node-dev-hook-19040724413613908.js:71:20)\n    at Object.nodeDevHook [as .ts] (/home/app/node_modules/ts-node-dev/lib/hook.js:63:13)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Module.require (node:internal/modules/cjs/loader:1005:19)"},"msg":"Oops!"}
{"level":60,"time":1641570141106,"pid":79,"hostname":"7677ae9f3529","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module._compile (/home/app/node_modules/source-map-support/source-map-support.js:568:25)\n    at Module.m._compile (/tmp/ts-node-dev-hook-19040724413613908.js:69:33)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at require.extensions.<computed> (/tmp/ts-node-dev-hook-19040724413613908.js:71:20)\n    at Object.nodeDevHook [as .ts] (/home/app/node_modules/ts-node-dev/lib/hook.js:63:13)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Module.require (node:internal/modules/cjs/loader:1005:19)"},"msg":"Oops!"}
{"level":60,"time":1641570150827,"pid":91,"hostname":"7677ae9f3529","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module._compile (/home/app/node_modules/source-map-support/source-map-support.js:568:25)\n    at Module.m._compile (/tmp/ts-node-dev-hook-19040724413613908.js:69:33)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at require.extensions.<computed> (/tmp/ts-node-dev-hook-19040724413613908.js:71:20)\n    at Object.nodeDevHook [as .ts] (/home/app/node_modules/ts-node-dev/lib/hook.js:63:13)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Module.require (node:internal/modules/cjs/loader:1005:19)"},"msg":"Oops!"}
{"level":60,"time":1641570169604,"pid":103,"hostname":"7677ae9f3529","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module._compile (/home/app/node_modules/source-map-support/source-map-support.js:568:25)\n    at Module.m._compile (/tmp/ts-node-dev-hook-19040724413613908.js:69:33)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at require.extensions.<computed> (/tmp/ts-node-dev-hook-19040724413613908.js:71:20)\n    at Object.nodeDevHook [as .ts] (/home/app/node_modules/ts-node-dev/lib/hook.js:63:13)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Module.require (node:internal/modules/cjs/loader:1005:19)"},"msg":"Oops!"}
{"level":60,"time":1641570982634,"pid":30,"hostname":"afe70a152351","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641571046175,"pid":42,"hostname":"afe70a152351","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
{"level":60,"time":1641571052830,"pid":31,"hostname":"afe70a152351","msg":"uncaughtException event","err":{"type":"Error","message":"Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/home/app/src/server.ts:30:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Module.m._compile (/home/app/node_modules/ts-node/src/index.ts:1371:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Object.require.extensions.<computed> [as .ts] (/home/app/node_modules/ts-node/src/index.ts:1374:12)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at main (/home/app/node_modules/ts-node/src/bin.ts:331:12)\n    at Object.<anonymous> (/home/app/node_modules/ts-node/src/bin.ts:482:3)"},"msg":"Oops!"}
